movement.h

```diff
@@ -0,0 +1,306 @@
+class movements{
+    public:
+    int motores[8];
+    int pwm[4];

```

Comment:
You should add a Motor Class and specify 4 attributes in this class, one for each motor. That will be a lot better than having this array. It cannot be tell which pins are for each motor.